x-pack/legacy/plugins/maps/public/index.scss

```diff
@@ -1,17 +1,3 @@
-// Import the EUI global scope so we can use EUI constants
-@import 'src/legacy/ui/public/styles/_styling_constants';
-
 /* GIS plugin styles */
 
-// Prefix all styles with "map" to avoid conflicts.
-// Examples
-//   mapChart
-//   mapChart__legend
-//   mapChart__legend--small
-//   mapChart__legend-isLoading
-
-@import './main';
-@import './mapbox_hacks';
-@import './connected_components/index';
-@import './components/index';
-@import '../../../../plugins/maps/public/layers/index';
+@import '../../../../plugins/maps/public/index';
```

Comment:
Did a very quick glance, I didn't verify all the new paths are correct, but no additional styles were added. So long as this import is correct 👍

Reply:
It is correct. Thanks @cchaos!